internal/host/validator.go

```diff
@@ -733,8 +732,8 @@ func (v *validator) printSufficientNetworkLatencyRequirementForRole(c *validatio
 
 func (v *validator) hasSufficientPacketLossRequirementForRole(c *validationContext) ValidationStatus {
 
-	if len(c.cluster.Hosts) == 1 {
-		// Single Node use case
+	if len(c.cluster.Hosts) == 1 || c.clusterHostRequirements.Total.PacketLossPercentage == nil || c.host.Role == models.HostRoleAutoAssign {
+		// Single Node use case || no requirements defined || role is auto assign (BZ-1974421)
```

Comment:
I usually mention BZ/tickets if it's a temporary fix, otherwise try to make the code as clear as possible by itself (or with some help of comments)

Reply:
understood, I'll remove the reference.